apps/avifenc.c

```diff
@@ -235,7 +235,7 @@ static void syntaxLong(void)
     printf("    -g,--grid MxN                     : Encode a single-image grid AVIF with M cols & N rows. Either supply MxN identical W/H/D images, or a single\n");
     printf("                                        image that can be evenly split into the MxN grid and follow AVIF grid image restrictions. The grid will adopt\n");
     printf("                                        the color profile of the first image supplied.\n");
-    printf("    -c,--codec C                      : AV1 codec to use (choose from versions list below)\n");
+    printf("    -c,--codec C                      : AV1/AV2 codec to use (choose from versions list below)\n");
```

Comment:
`AVIF_CODEC_AVM` is an experimental feature, so I would avoid mentioning AV2 if `AVIF_CODEC_AVM` is OFF.

Suggestion:
```c
printf(" -c,--codec C       : codec to use (choose from versions list below)\n");
```